frontend/src/components/document/DocumentView/StyledText.jsx

```diff
@@ -1,6 +1,6 @@
 import { getCommentThreadsOnTextNode } from "@/util/editorCommentUtils";
 import { getRedactionsOnTextNode, getMarkFromLeaf } from "@/util/editorRedactionUtils";
-import CommentedText from "@/components/document/Document/CommentedText";
+import CommentedText from "@/components/document/DocumentView/CommentedText";
```

Comment:
This was giving the error: 
`Failed to load url ...`

Reply:
woops ty for fix